openkore-all

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+#source var/function
+. ./checkfiles.sh
+PATH=./:$PATH
+YELLOW='\033[1;33m'
+GREEN='\033[0;32m'
+NOCOLOR='\033[0m'
+RED='\033[0;31m'
+
+print_start() {
+	echo "Welcome to Openkore Linux"
+	echo "This script is under GNU v3, is free, anyone can use and modify it."
+	echo "Thanks to Openkore developers and contributors."
+	sleep 1.5
+	echo "${YELLOW}Checking for files if everything is in order..."
+}
+
+run_screens () {
+	read -p 'Please enter the number of bots you want to run: ' bot_number
+	echo "Setting up screen sessions..."
+	i=0
+	while [ "$i" -ne $bot_number ]
+		do
+		screen -dmS bot$i
+		echo "Screen Creating bot$i"
+		sleep 0.5
+		i=$(( i + 1 ))
+	done
+	echo "${GREEN}Screen sessions were created successfully!"
+}
+
+run_openkores () {
+	echo "${NOCOLOR}Now Openkore will execute in each screen session."
+	echo "This may take a while deppending on how many bots are you running."
+	echo "${YELLOW}Please do not touch until the process has finished.${NOCOLOR}"
+	sleep 2
+	echo "Openkore will start in each session in 5 seconds"
+	sleep 5
+	echo "Starting..."
+	i=0
+	while [ "$i" -ne $bot_number ]
+		do
+		screen -S bot$i -X stuff 'perl ./openkore.pl --control=control'$i' --logs=logs/bot'$i'\n'
```

Comment:
This goes against existing OpenKore conventions. The recommended way to run multiple bots using different configurations is with the profiles plugin. If you want this script to be included in the project, it should be compatible with the profiles plugin instead of creating multiple control folders.

http://openkore.com/index.php/Profiles